### DISCUSSION TITLE
Mousewheel works bad on windows

### DISCUSSION THREAD

**Discussion by maxraywork** [nolimits4web/swiper#5371] [Q&A]
My page: https://kromad.com/test

Try to scroll inside the swiper and you will see, that it just ignore your scroll after first or last slide and sometimes it works. 
You can compare it with the swiper mousewheel demo: https://codesandbox.io/s/htgn9

It's bad in all browsers on Windows 10. On mac with trackpad it's fine.

**Comment by rtabulov** [comment]
cannot really help you without seeing your code
make sure you install modules that you need (like Mousewheel and others) via Swiper.use

```js
import { Swiper, Mousewheel } from 'swiper';
Swiper.use([Mousewheel]);
```
OR use a bundle
```js
import Swiper from 'swiper/bundle';
```
OR if you use cdn, load a bundle too
```html
<script src="https://unpkg.com/swiper@7/swiper-bundle.min.js"></script>
```

**Reply by maxraywork** [reply]
I use cdn and elementor. Here is the swiper init code in the js file
![image](https://user-images.githubusercontent.com/73395911/149938532-88a492a1-0819-434a-b008-66844d10d7be.png)
